### PR TITLE
merge Release/live service/55.1 into master

### PR DESCRIPTION
**Pull request by jpotter-DFE**

### CODE REVIEW

File: DFC.App.JobGroups.Services.CacheContentService/SharedContentCacheReloadService.cs [hunk 1]
```diff
@@ -10,29 +10,31 @@
...
+using DFC.Common.SharedContent.Pkg.Netcore.Interfaces;
 
 namespace DFC.App.JobGroups.Services.CacheContentService
 {
     public class SharedContentCacheReloadService : ISharedContentCacheReloadService
     {
         private readonly ILogger<SharedContentCacheReloadService> logger;
         private readonly IMapper mapper;
-        private readonly IDocumentService<ContentItemModel> contentItemDocumentService;
+        //private readonly IDocumentService<ContentItemModel> contentItemDocumentService;
```
Comment (hnandra) [inline DFC.App.JobGroups.Services.CacheContentService/SharedContentCacheReloadService.cs:21]: Delete comment

File: DFC.App.JobGroups/Controllers/PagesController.cs [hunk 2]
```diff
@@ -1,41 +1,54 @@
...
 using System.Threading.Tasks;
+using Constants = DFC.Common.SharedContent.Pkg.Netcore.Constant.ApplicationKeys;
 
 namespace DFC.App.JobGroups.Controllers
 {
     public class PagesController : Controller
     {
         public const string RegistrationPath = "job-groups";
         public const string LocalPath = "pages";
+        //public const string SharedContentStaxId = "2c9da1b3-3529-4834-afc9-9cd741e59788";
```
Comment (hnandra) [inline DFC.App.JobGroups/Controllers/PagesController.cs:24]: Delete comment

**Review by hnandra — CHANGES_REQUESTED** [review]
Delete comments

**Review by hnandra — DISMISSED** [review]

**Review by gcompton-DFE — DISMISSED** [review]
Reviewed

**Review by zlu-DFE — APPROVED** [review]

**Review by hnandra — APPROVED** [review]

**Review by gcompton-DFE — APPROVED** [review]
Reviewed

**Pull request merged**